.npmrc

```diff
@@ -2,5 +2,5 @@ legacy-peer-deps=true # TODO: remove this line in v4.0
 strict-ssl=false # Fix problem when downloading cypress binary
 ca = ''
 unsafe-perm=true
-engine-strict=true
+engine-strict=false
```

Comment:
Why do we need to change it ?

Reply:
<img width="993" alt="image" src="https://github.com/user-attachments/assets/7e838b7a-49ca-42e8-9568-7463cd3f011c">

Reply:
Should we just update CI Node version instead ?